src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -381,7 +403,13 @@ public function rsvp_process() {
 			wp_send_json_error( $response );
 		}
 
-		$products = (array) tribe_get_request_var( 'product_id' );
+		$products = [];
+
+		if ( isset( $_POST['tribe_tickets'] ) ) {
+			$products = wp_list_pluck( $_POST['tribe_tickets'], 'ticket_id' );
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['tribe_tickets']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 3fab0e02d123299012616c67a1ee85e6</sub>